cmd/zdb/zdb.c

```diff
@@ -4477,8 +4498,10 @@ main(int argc, char **argv)
 		args.path = searchdirs;
 		args.can_be_active = B_TRUE;
 
-		error = zpool_tryimport(g_zfs, target, &cfg, &args);
+		error = zpool_tryimport(g_zfs, target_pool, &cfg, &args);
```

Comment:
Is there any way we can use the error message set by `zfs_error_aux()` in `zpool_tryimport()` to provide a more descriptive message here?

```c
4568	if (error)
4569	 	fatal("can't open '%s': %s", target, strerror(error));
```

Reply:
Let's leave this for later.